modules/ROOT/pages/release-notes/release-notes53.adoc

```diff
@@ -33,8 +34,8 @@ This new default setting has been added to improve the user experience for the e
 
 For information on:
 
-* The `toolbar_location` option, see: link:{baseurl}/configure/editor-appearance/#toolbar_location[User interface options - toolbar_location].
-* {productname} in inline mode, see: link:{baseurl}/general-configuration-guide/use-tinymce-inline/#enablinginlineeditingmode[Setup inline editing mode].
+* The `toolbar_location` option, see: xref:configure/editor-appearance.adoc#toolbar_location[User interface options - toolbar_location].
```

Comment:
This xref doesn't resolve as the `toolbar_location` id hasn't been added to the heading in `toolbar_location.adoc`. e.g:
```
[[toolbar_location]]
== `toolbar_location`
```

Reply:
done